docs/web/sdk-components-styling.mdx

```diff
@@ -116,20 +126,189 @@ export default function ComponentWithIcon(props) {
+
+<Tabs
+  defaultValue="icon"
+  values={[
+    { label: 'Icon', value: 'icon', },
+    { label: 'Registration', value: 'registration', },
+    { label: 'Component With Icon', value: 'component', },
+    { label: 'Layout', value: 'layout', },
+    { label: 'App Config', value: 'config', },
+    { label: 'UI', value: 'ui', },
+  ]
+}>
+<TabItem value="icon">
+
+```tsx title="src/icons/CustomIcon.svg"
+import createSvgIcon from "@vertigis/web/ui/icons/utils/createSvgIcon";
+import React from "react";
+
+export default createSvgIcon(
+    <path d="M20 11H7.8l5.6-5.6L12 4l-8 8 8 8 1.4-1.4L7.8 13H20v-2z" />
+);
+```
+
+</TabItem>
+<TabItem value="registration">
+
+```ts title="src/index.ts"
+import PointsOfInterest, {
+    PointsOfInterestModel,
+} from "./components/PointsOfInterest";
+import { LibraryRegistry } from "@vertigis/web/config";
+import ComponentWithIcon from "./components/PointsOfInterest/ComponentWithIcon/ComponentWithIcon";
+import ComponentWithIconModel from "./components/PointsOfInterest/ComponentWithIcon/ComponentWithIconModel";
+import CustomIcon from "./icons/CustomIcon";
+
+const LAYOUT_NAMESPACE = "my.custom.namespace";
+
+export default function (registry: LibraryRegistry): void {
+    registry.registerComponent({
+        name: "component-with-icon",
+        namespace: LAYOUT_NAMESPACE,
+        getComponentType: () => ComponentWithIcon,
+        itemType: "component-with-icon-model",
+        title: "Component With Icon",
+    });
+    registry.registerModel({
+        getModel: (config) => new ComponentWithIconModel(config),
+        itemType: "component-with-icon-model",
+    });
+    // highlight-start
+    registry.registerIcon({
+        id: "custom-arrow",
+        getComponentType: () => CustomIcon,
+    });
+    // highlight-end
+}
+```
+
+</TabItem>
+<TabItem value="component">
+
+```tsx title="src/components/ComponentWithIcon/ComponentWithIcon.tsx"
+import React, { FC } from "react";
+import {
+    LayoutElement,
+    LayoutElementProperties,
+} from "@vertigis/web/components";
+import DynamicIcon from "@vertigis/web/ui/DynamicIcon";
+import ComponentWithIconModel from "./ComponentWithIconModel";
+import MenuItem from "@vertigis/web/ui/MenuItem";
+import IconButton from "@vertigis/web/ui/IconButton";
+import ListItemText from "@vertigis/web/ui/ListItemText";
+
+const ComponentWithIcon: FC<LayoutElementProperties<ComponentWithIconModel>> = (
+    props
+) => {
+    return (
+        <LayoutElement {...props} style={{ backgroundColor: "white" }}>
+            <MenuItem>
+                <ListItemText primary="Custom Arrow Icon" />
+                <IconButton title="Custom Arrow Icon">
+                    // highlight-next-line
+                    <DynamicIcon src="custom-arrow" />
```

Comment:
Similar to my other comment, let's use DynamicIcon in a dynamic way, instead of with a hard-coded string.

Reply:
Done.